### PR TITLE
added the termination date and time as motd

### PR DESCRIPTION
**Pull request by arbabnazar**

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @arbabnazar! I've created [OSPR-352](https://openedx.atlassian.net/browse/OSPR-352) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://code.edx.org/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

**Comment by sarina** [comment]
@arbabnazar - we have your work under @arbab-edx. Should we add this account too?

**Comment by arbabnazar** [comment]
@sarina, can you please add the arbabnazar account and delete the arbab-edx please. Thanks

**Comment by sarina** [comment]
Sure, will-do. Thanks!

**Comment by arbabnazar** [comment]
@e0d, can you please look into the added role. Thanks

File: playbooks/roles/edx-sandbox/templates/etc/update-motd.d/temiate_motd.j2 [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ec2 = boto.connect_ec2()
+instance_id = get_instance_metadata()['instance-id']
+reservations = ec2.get_all_instances(instance_ids=[instance_id])
+
+try:
+    terminate_time = datetime.strptime(str(reservations[0].instances[0].tags['instance_termination_time']), "%m-%d-%Y %H:%M:%S")
+    print ("This system will be terminated on {:%b %d, %Y} at {:%H:%M:%S} UTC".format(terminate_time,terminate_time))
+
+except KeyError:
+    print ("This system will be terminated after the 7 Days of its Creation")
```
Comment (e0d) [inline playbooks/roles/edx-sandbox/templates/etc/update-motd.d/temiate_motd.j2]: This is pseudo python

```
  instance = reservations[0].instances[0]
  if instance.tags['instance_termination_time']:
    terminate_time = instance.tags['instance_termination_time']
  else:
    termination_time = instance.get_launch_time + 7d.
```

This way the message can be the same in either case.

**Comment by e0d** [comment]
@feanil or @maxrothman I'm happy with this as is, second review.

**Comment by feanil** [comment]
:+1:  please squash before merging.

**Comment by e0d** [comment]
:+1:

**Pull request merged**